test/WebSites/MvcTagHelpersWebSite/Components/ProductsViewComponent.cs

```diff
@@ -16,7 +16,11 @@ public IViewComponentResult Invoke(string category)
         {
             IExpirationTrigger trigger;
             var products = ProductsService.GetProducts(category, out trigger);
-            EntryLinkHelpers.ContextLink.AddExpirationTriggers(new[] { trigger });
+
+            if (EntryLinkHelpers.ContextLink != null)
```

Comment:
Shouldn't it be available hanging off of `MemoryCache`? `MemoryCache.CreateLinkingScope()` -> `MemoryCache.CurrentLinkingScope`?